pipeline/4/Jenkinsfile-tier-0.groovy

```diff
@@ -0,0 +1,73 @@
+/*
+    Pipeline script for executing Tier 0 test suites for RH Ceph 4.x.
+*/
+// Global variables section
+def nodeName = "centos-7"
+def cephVersion = "nautilus"
+def sharedLib
+def testStages = ['sanity_rpm': {
+                    stage('RPM Sanity suite') {
+                        script {
+                            withEnv([
+                                "sutConf=conf/${cephVersion}/ansible/sanity-ceph-ansible.yaml",
+                                "testSuite=suites/${cephVersion}/ansible/sanity_ceph_ansible.yaml",
+                                "addnArgs=--post-results --log-level DEBUG"
+                            ]) {
+                                sharedLib.runTestSuite()
+                            }
+                        }
+                    }
+                 }, 'sanity_containerized': {
+                    stage('Containerized Sanity suite') {
+                        script {
+                            withEnv([
+                                "sutConf=conf/${cephVersion}/ansible/sanity-ceph-ansible.yaml",
+                                "testSuite=suites/${cephVersion}/ansible/sanity_containerized_ceph_ansible.yaml",
+                                "addnArgs=--post-results --log-level DEBUG"
+                            ]) {
+                                sharedLib.runTestSuite()
+                            }
+                        }
+                    } 
+                 }
+                 ]
+
+// Pipeline script entry point
+node(nodeName) {
+
+    timeout(unit: "MINUTES", time: 30) {
+        stage('Install prereq') {
+            checkout([
+                $class: 'GitSCM',
+                branches: [[name: '*/4.x_build']],
+                doGenerateSubmoduleConfigurations: false,
+                extensions: [[
+                    $class: 'SubmoduleOption',
+                    disableSubmodules: false,
+                    parentCredentials: false,
+                    recursiveSubmodules: true,
+                    reference: '',
+                    trackingSubmodules: false
+                ]],
+                submoduleCfg: [],
+                userRemoteConfigs: [[url: 'https://github.com/manasagowri/cephci.git']]
```

Comment:
shouldn't this be `https://github.com/red-hat-storage/cephci`. you are using your fork here.